src/Http/Http.Results/test/EmptyResultTest.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Http.Result;
+
+public class EmptyResultTest
+{
+    [Fact]
+    public async Task EmptyResult_DoesNothing()
+    {
+        // Arrange
+        var emptyResult = EmptyResult.Instance;
+
+        // Act
+        var httpContext = GetHttpContext();
+        var memoryStream = httpContext.Response.Body;
+        await emptyResult.ExecuteAsync(httpContext);
+
+        // Assert
+        Assert.Equal(StatusCodes.Status200OK, httpContext.Response.StatusCode);
+        Assert.Equal(memoryStream, httpContext.Response.Body);
```

Comment:
```suggestion
        var memoryStream = (MemoryStream)httpContext.Response.Body;
        await emptyResult.ExecuteAsync(httpContext);

        // Assert
        Assert.Equal(StatusCodes.Status200OK, httpContext.Response.StatusCode);
        Assert.Equal(0, memoryStream.Length);
```

It might be slightly better to just make the `Response.Body` stream a parameter to `GetHttpContext(Stream)`. That way you could avoid casting back to `MemoryStream`, but it's not a big deal either way.